main.nf

```diff
@@ -81,17 +83,24 @@ if (!seq_date) {
   throw new Exception("${major_release} release not supported in map variables in nf code.")
 }
 
-// If major release is not TEST
-// Specify production synapse id to pass into processing
-if (major_release != "TEST") {
-  project_id = "syn3380222"
-  center_map_synid = "syn10061452"
-  is_prod = true
-}
-else {
+// define whether to run TEST, STAGING or PRODUCTION
+if (major_release == "TEST") {
   project_id = "syn7208886"
   center_map_synid = "syn11601248"
   is_prod = false
+  is_staging = false
+} else if (major_release == "STAGING"){
+  project_id = "syn22033066"
+  center_map_synid = "syn22089188"
+  is_prod = false
+  is_staging = true
+}
```

Comment:
nit: Is there any automatic linting tools for `.nf` files? I noticed there is some inconsistent spacing, new lines, ect... that could use some automatic linting.

If there is, maybe we can also add it into a precommit too

Reply:
unfortunately it's pretty difficult:
- `nf-core` repo has [linting](https://nf-co.re/docs/contributing/guidelines/requirements/linting) but they don't have pre-commit available for it, so would need to install entire`nf-core` as a dependency and it may not be possible to skip all structural/nf-core specific linting tests (because they have certain expectations for nf-core pipelines that we don't necessarily follow or need or want to maintain)
- [linter-rules-for-nextflow](https://github.com/awslabs/linter-rules-for-nextflow) has linting, but we'd probably need to implement a module for it as it's easiest to run through their docker image + I tried the docker image and it doesn't seem to run on `main.nf` files/doesn't seem like it looks at new lines, inconsistent spacing

**Update:** Based on this: https://github.com/nf-core/tools/issues/1642, it seems like a formatter for just nextflow scripts is still in development :'). Can still add basic formatting like new-lines, etc.